docs/diagrams/StorageClassDiagram.puml

```diff
@@ -1,24 +1,28 @@
-@startuml
+@startuml StorageClassDiagram
 !include style.puml
 skinparam arrowThickness 1.1
 skinparam arrowColor STORAGE_COLOR
 skinparam classBackgroundColor STORAGE_COLOR
 
 Interface Storage <<Interface>>
 Interface UserPrefsStorage <<Interface>>
-Interface AddressBookStorage <<Interface>>
+Interface FitBizStorage <<Interface>>
 
 Class StorageManager
 Class JsonUserPrefsStorage
-Class JsonAddressBookStorage
+Class JsonFitBizStorage
+Class StorageReaderWriter
 
 StorageManager .left.|> Storage
 StorageManager o-right-> UserPrefsStorage
-StorageManager o--> AddressBookStorage
+StorageManager o--> FitBizStorage
 
 JsonUserPrefsStorage .left.|> UserPrefsStorage
-JsonAddressBookStorage .left.|> AddressBookStorage
-JsonAddressBookStorage .down.> JsonSerializableAddressBookStorage
-JsonSerializableAddressBookStorage .right.> JsonSerializableClient
-JsonSerializableClient .right.> JsonAdaptedTag
-@enduml
+FitBizStorage <|.right. JsonFitBizStorage
+JsonFitBizStorage .down.> JsonSerializableFitBizStorage
+JsonSerializableFitBizStorage .right.> JsonSerializableClient
+JsonSerializableClient -right-> "*" JsonAdaptedTag
+JsonSerializableClient -right-> "*" JsonAdaptedSport
+JsonSerializableClient -up-> "*" JsonAdaptedExercise
+JsonSerializableClient -down-> "*" JsonAdaptedSchedule
```

Comment:
I don't think we have a `JsonSerializableClient` class. The original AB3 diagram might be erroneous.